content/articles/one-way-anova-excel-testing-normality-homogeneity-using-python/index.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /one-way-anova-excel-testing-normality-homogeneity-using-python/
+title: Solving a One Way ANOVA with Excel using Python
+description: In this article we will be creating an ANOVA table using Microsoft Excel, box plots to test the normality of a one way ANOVA, and Bartlett's rule for homogeneity with Python.
+author: kuteyi-victor-toluwase
+date: 2021-12-02T00:00:00-17:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/one-way-anova-excel-testing-normality-homogeneity-using-python/hero.jpg
+    alt: One Way ANOVA with Excel using Python example image
+---
+Over the years, data has been an essential part of how the world works. These data range from GDP to blood samples to every aspect of the world. As data grew, statistics found ways to extract meaning from them. 
+<!--more-->
+One of these methods is the analysis of variance (ANOVA). ANOVA is a set of statistical models that analyses the differences among means. Microsoft came up with statistical add-ins within Excel to solve some of these problems. Although Microsoft Excel is still rather limited. Later came tools like R-studio and Python. 
+
+With Python, the test for normality and homogeneity became easier. This tutorial will focus on creating an ANOVA table using Microsoft Excel, box plots to test the normality of a one way ANOVA, and Bartlett's rule for homogeneity with Python.
+
+### Prerequisites
+This tutorial requires readers to have proper knowledge on the following:
+- Here is a link to the [Analysis of variance (ANOVA)](https://www.investopedia.com/terms/a/anova.asp) documentation.
```

Comment:
- Analysis of variance (ANOVA). Here's a [link]((https://www.investopedia.com/terms/a/anova.asp)) to the documentation.